app/views/students/index.html.erb

```diff
@@ -10,10 +10,13 @@
     <% @students.each do |student| %>
     <li> <%= student.name %></li>
      <%= link_to 'Edit', edit_student_path(student) %>
+     <%= link_to 'Delete', student, method: :destroy, data: {confirm: 'Are you sure?'} %>
```

Comment:
One last thing is that, confusingly, the method here has to be `:delete` not `:destroy`. **destroy** is the action, **DELETE** is the request type, like GET, POST, PUT, PATCH, etc. They get used interchangeably a lot but they are not the same.